hazelcast/src/main/java/com/hazelcast/config/WanReplicationRefReadOnly.java

```diff
@@ -27,15 +29,18 @@ public WanReplicationRefReadOnly(WanReplicationRef ref) {
         super(ref);
     }
 
+    @Override
     public WanReplicationRef setName(String name) {
         throw new UnsupportedOperationException("This config is read-only");
     }
 
+    @Override
     public WanReplicationRef setMergePolicy(String mergePolicy) {
         throw new UnsupportedOperationException("This config is read-only");
     }
 
-    public WanReplicationRef setFilter(String mergePolicy) {
+    @Override
+    public WanReplicationRef setFilters(List<String> filters) {
```

Comment:
Due to the missing `@Override` we missed that the old `setFilter()` wasn't existing anymore and we to add this new one.